apps/cookbook/src/app/examples/modal-example/embedded-modal-example/embedded-modal-example.component.html

```diff
@@ -21,6 +21,13 @@
   ></cookbook-modal-example-configuration>
 </cookbook-example-configuration-wrapper>
 
+<kirby-checkbox
+  *ngIf="alertBeforeClose"
+  attentionLevel="1"
+  [checked]="disableAlertBeforeClose"
+  text="Disable alert before closing modal"
+  (checkedChange)="disableAlertBeforeClose = $event"
```

Comment:
Consider flipping this so it aligns with the config checkbox (i.e. alert if checked):
```suggestion
  *ngIf="alertBeforeClose"
  attentionLevel="1"
  [checked]="alertBeforeClose"
  text="Alert before closing modal"
  (checkedChange)="alertBeforeClose = $event"
```

Reply:
@jakobe I have done a slight variation of your suggestion. See the `_toggleAlertBeforeClose` method in `embedded-modal-example.component.ts`. This makes it possible to enable the alert from within the modal and not just the configuration menu before opening the modal.

Reply:
@mark-drastrup Great 👍🏻 
We do use the modal example configuration for everything else, so maybe move the checkbox to that one? 🤔 
Also see my other comment about simplifying the check for whether to alert or not.

Reply:
@mark-drastrup I don't think this got resolved? What I mean is this:
![image](https://github.com/kirbydesign/designsystem/assets/9556693/27394350-b3f0-470e-9fee-a6e36cd0dd02)

Reply:
Hmm I did make the change, but for some reason the tests and deployment got stuck when I pushed the last batch of feedback, so the changes were never reflected 🤔 I'm pushing the next batch in a couple of minutes and will let you know, when they changes are live 😄